osm-fudge/amenity-example.py

```diff
@@ -0,0 +1,26 @@
+"""
```

Comment:
If you put a `#!/usr/bin/env python3` line here we can make this script executable. Look into so called "shebangs".

Reply:
Yes right 👍